MobileFilter/sections/specific_web.txt

```diff
@@ -80,6 +80,8 @@ sp.manga.nicovideo.jp##div[style^="--ads-width:"]
 !
 ! SECTION: Regular rules
 !
+superporn.com##.intensive-add
```

Comment:
It's already in the AdGuard Base.

Reply:
Adguard mobile ads is default in ublock for mobile users

Reply:
ok will remove